src/main.c

```diff
@@ -48,6 +48,8 @@ char __noinit __stack cdcacm_setup_stack_area[CDCACM_STACKSIZE];
 char __noinit __stack baudrate_reset_stack_area[RESET_STACKSIZE];
 char __noinit __stack usb_serial_stack_area[USBSERIAL_STACKSIZE];
 
+const char *firmware_version="CODK-M_1.6.0";
```

Comment:
Would be nice to ping on Martino to let him know that we are tagging the x86 version number at the end of binary image.  Most likely he will have to read a small chunk of data from the flash at an address near the end of file and "grep" for the version string expected.